packages/ui/src/theme/css/component/table.scss

```diff
@@ -0,0 +1,121 @@
+  /**
+   * Default Table primitive stylings
+   */
+  border-collapse: collapse;
+  width: 100%;
+  display: table;
+  box-sizing: border-box;
+  text-indent: initial;
+
+  .amplify-table__head {
+    display: table-header-group;
+    vertical-align: middle;
+    border-color: inherit;
+  }
+
+  .amplify-table__body {
+    display: table-row-group;
+    vertical-align: middle;
+    border-color: inherit;
+  }
+
+  .amplify-table__foot {
+    display: table-footer-group;
+    vertical-align: middle;
+    border-color: inherit;
+  }
+
+  .amplify-table__row {
+    display: table-row;
+    vertical-align: inherit;
+    border-color: inherit;
+  }
+
+  .amplify-table__th,
+  .amplify-table__td {
+    display: table-cell;
+    vertical-align: inherit;
+    border-color: var(--amplify-components-table-border-color);
+    border-style: var(--amplify-components-table-border-style);
+    border-width: var(--amplify-components-table-border-width);
+    padding: var(--amplify-components-table-padding);
+    color: var(--amplify-components-table-color);
+    font-size: var(--amplify-components-table-font-size-default);
+  }
+
+  .amplify-table__th {
+    font-weight: var(--amplify-components-table-font-weight-th);
+  }
+
+  .amplify-table__td {
+    font-weight: var(--amplify-components-table-font-weight-td);
+  }
+
+  .amplify-table caption {
+    color: var(--amplify-components-table-color);
+    font-size: var(--amplify-components-table-font-size-default);
+    word-break: break-all;
+  }
+
+  .amplify-table__summary {
+    color: var(--amplify-colors-font-secondary);
+    font-size: var(--amplify-components-table-font-size-summary);
+    font-style: italic;
+  }
+
+  /**
+   * Data attribute stylings
+   */
+  &:not([data-variation='bordered']) {
+    .amplify-table__td:not(:first-child),
+    .amplify-table__th:not(:first-child) {
+      border-left: none;
+    }
+
+    .amplify-table__td:not(:last-child),
+    .amplify-table__th:not(:last-child) {
+      border-right: none;
+    }
+  }
+
+  &[data-variation='striped'] {
+    .amplify-table__body .amplify-table__row:nth-child(odd) {
+      background-color: var(--amplify-colors-background-secondary);
+    }
+  }
+
+  &[data-highlightonhover='true'] {
+    .amplify-table__body .amplify-table__row:hover {
+      background-color: var(--amplify-colors-background-tertiary);
+    }
+  }
+
+  &[data-size='small'] {
+    .amplify-table__th,
+    .amplify-table__td {
+      font-size: var(--amplify-components-table-small-font-size);
+      padding: var(--amplify-components-table-small-padding);
+    }
+
+    caption {
+      font-size: var(--amplify-components-table-small-font-size);
+    }
+  }
+
+  &[data-size='large'] {
+    .amplify-table__th,
+    .amplify-table__td {
+      font-size: var(--amplify-components-table-large-font-size-default);
+      padding: var(--amplify-components-table-large-padding);
+    }
+
+    caption {
+      font-size: var(--amplify-components-table-large-font-size-default);
```

Comment:
Why include `default` in `--amplify-components-table-large-font-size-default`?

Reply:
yea, we don't need the `default` technically to be consistent with other custom variables in other primitives